src/calc.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const { IscalculatingRight } = require('./IscalculatingRight');
+
+const calc = (userInput, digits) => {
+  const currentNumber = digits;
```

Comment:
The variable 'currentNumber' is not necessary, as you can directly use 'digits' in the rest of the function.